tests/Unit/Evolution/DiscontinuousGalerkin/Initialization/Test_Mortars.cpp

```diff
@@ -0,0 +1,293 @@
+      std::make_pair(Direction<1>::upper_xi(), east_id);
+  const MortarMap<1, Mesh<0>> expected_mortar_meshes{{boundary_mortar_id, {}},
+                                                     {interface_mortar_id, {}}};
+  const MortarMap<1, std::array<Spectral::MortarSize, 0>> expected_mortar_sizes{
+      {boundary_mortar_id, {}}, {interface_mortar_id, {}}};
+
+  test_impl(initial_extents, element, time_step_id, expected_mortar_meshes,
+            expected_mortar_sizes);
+}
+
+template <>
+void test<2>() {
+  INFO("2D");
+  // Reference element is denoted by X, has two internal boundaries (east and
+  // south) and two external boundaries (west and north):
+  //
+  // ^ eta
+  // +-+-+> xi
+  // |X| |
+  // +-+-+
+  // | | |
+  // +-+-+
+
+  const ElementId<2> element_id{0, {{{1, 0}, {1, 1}}}};
+  const ElementId<2> east_id(0, {{SegmentId{1, 1}, SegmentId{1, 1}}});
+  const ElementId<2> south_id(0, {{SegmentId{1, 0}, SegmentId{1, 0}}});
+  const std::vector initial_extents{std::array{3_st, 2_st}};
+
+  DirectionMap<2, Neighbors<2>> neighbors{};
+  neighbors[Direction<2>::upper_xi()] = Neighbors<2>{{east_id}, {}};
+  neighbors[Direction<2>::lower_eta()] = Neighbors<2>{{south_id}, {}};
+
+  const Element<2> element{element_id, neighbors};
+  const TimeStepId time_step_id{true, 3, Time{Slab{0.2, 3.4}, {3, 100}}};
+
+  // We are working with 4 mortars here: the domain boundary west and north,
+  // and interfaces south and east.
+  const auto boundary_mortar_id_west = std::make_pair(
+      Direction<2>::lower_xi(), ElementId<2>::external_boundary_id());
+  const auto boundary_mortar_id_north = std::make_pair(
+      Direction<2>::upper_eta(), ElementId<2>::external_boundary_id());
+  const auto interface_mortar_id_east =
+      std::make_pair(Direction<2>::upper_xi(), east_id);
+  const auto interface_mortar_id_south =
+      std::make_pair(Direction<2>::lower_eta(), south_id);
+
+  const MortarMap<2, Mesh<1>> expected_mortar_meshes{
+      {boundary_mortar_id_west, Mesh<1>(2, Spectral::Basis::Legendre,
+                                        Spectral::Quadrature::GaussLobatto)},
+      {boundary_mortar_id_north, Mesh<1>(3, Spectral::Basis::Legendre,
+                                         Spectral::Quadrature::GaussLobatto)},
+      {interface_mortar_id_east, Mesh<1>(2, Spectral::Basis::Legendre,
+                                         Spectral::Quadrature::GaussLobatto)},
+      {interface_mortar_id_south, Mesh<1>(3, Spectral::Basis::Legendre,
+                                          Spectral::Quadrature::GaussLobatto)}};
+  MortarMap<2, std::array<Spectral::MortarSize, 1>> expected_mortar_sizes{};
+  for (const auto& mortar_id_and_mesh : expected_mortar_meshes) {
+    expected_mortar_sizes[mortar_id_and_mesh.first] = {
+        {Spectral::MortarSize::Full}};
+  }
+
+  test_impl(initial_extents, element, time_step_id, expected_mortar_meshes,
+            expected_mortar_sizes);
+}
+
+template <>
+void test<3>() {
+  INFO("3D");
```

Comment:
I know you can't draw ascii art here as easily, but could you at least write comments (just like in the 1d, 2d cases) to describe what the setup of the element and the mortars is?